web/templates/pages/project_detail.html

```diff
@@ -3,6 +3,8 @@
 {% block content%}
 	<div class="row">
 		<h3><a href="{% url 'web.views.project_detail' project_id=project.project_id %}">{{ project.project_name }}</a></h3>
+		<a class="btn btn-default" href="{% url 'web.views.project_new' %}" class="top-menu"><span class="glyphicon glyphicon-plus"></span></a>
```

Comment:
You did not name your URL and now you have this oogly peace of :hankey:  here. Please name the URL, and change this.